R/funding-element.R

```diff
@@ -15,58 +18,76 @@
 #' add_funding(funder_name = "National Science Foundation",
 #'             funder_identifier = "http://dx.doi.org/10.13039/100000001",
 #'             award_number = "1656026",
-#'             award_title = "LTER: Beaufort Sea Lagoons: An Arctic Coastal Ecosystem in Transition",
+#'             award_title = "National Science Foundation Grant",
 #'             award_url = "https://www.nsf.gov/awardsearch/showAward?AWD_ID=1656026",
 #'             funding_description = "BLE LTER is supported by the National Science 
 #'                                    Foundation under award #1656026 
 #'                                    (2017-08-01 to 2022-07-31)." )
 #' @export
 
-add_funding <- function(funder_name, funder_identifier, award_number,
-                        award_title, award_url = NULL, funding_description = NULL)  {
+add_funding <- function(CVPIA_default_funder = NULL, funder_name, funder_identifier = NULL, 
```

Comment:
i don't like the argument name `CVPIA_default_funder`. I think you could just use `funder_name` only and if someone provides one from the list common funders than it pops out the intended info

Reply:
Okay will do